File: vbnet/UnicodeFileToHtmlTextConverter.Tests/UnicodeFileToHtmlTextConverterTests.vb
Imports Xunit

Namespace UnicodeFileToHtmlTextConverter.Tests
    Public Class UnicodeFileToHtmlTextConverterTests
        <Fact>
        Public Sub Foobar()
            Dim converter = New UnicodeFileToHtmlTextConverter("foobar.txt")
            Assert.Equal("fixme", converter.GetFilename())
        End Sub
    End Class
End Namespace

File: perl/test/UnicodeFileToHtmlTextConverter/UnicodeFileToHtmlTextConverterTests.pl
use strict;
use warnings;

use Test::More 0.96;

use lib 'lib/UnicodeFileToHtmlTextConverter';

require UnicodeFileToHtmlTextConverter;

subtest 'foobar' => sub {
    my $converter = new UnicodeFileToHtmlTextConverter("test/UnicodeFileToHtmlTextConverter/foobar.txt");

    is($converter->getFileName(), "fixme");
};

done_testing();